torch/ao/quantization/pt2e/qat_utils.py

```diff
@@ -771,63 +822,27 @@ def _fold_conv_bn_qat_helper(m: GraphModule, is_cuda: bool) -> GraphModule:
     _remove_extra_dequantize(m)
 
     for r in replacements:
-        node_map = _get_conv_bn_getitem_add_relu_nodes(r)
-        (_, conv_node) = node_map["conv"]
-        (_, bn_node) = node_map["bn"]
+        node_map = _get_conv_bn_pattern_nodes(r)
 
         # Step (2): Copy over metadata from original subgraph
         for original_node, replacement_node in node_map.values():
             replacement_node.meta = original_node.meta
 
-        # Step (3): Fold BN weights into conv
+        # Step (3): Copy over args for weight (and optionally bias) q - dq nodes
+        _copy_over_q_dq_args(*node_map["conv_weight_q"])
```

Comment:
I'm wondering if these can be simplified if we use these? https://github.com/pytorch/pytorch/blob/main/torch/ao/quantization/pt2e/utils.py#L291

Reply:
Yeah, we can explore that as a BE task along with the conv arg copying. I prefer to do that separately